recipes/autopxd2/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "autopxd2" %}
+{% set version = "1.1.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/autopxd2-{{ version }}.tar.gz
+  sha256: faa4b7eeb6b1d7217c46d3660640d4a72fb5c86d4559017c1f58a55d34332b2a
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - autopxd = autopxd:cli
+
+requirements:
+  host:
+    - pip
+    - python >=2.7,<3 | >=3.4
```

Comment:
Based on [this text in the README]( https://github.com/gabrieldemarmiesse/python-autopxd2/tree/v1.1.0#python-autopxd2 )